doc/decompressor_permissive.md

```diff
@@ -58,3 +58,24 @@ This behavior has no consequence for the rest of the frame decoding process.
 
 In newer versions, the 2 reserved bits are actively checked for value zero,
 and the decoder reports a corruption error if they are not.
+
+
+
+Truncated Huffman states
```

Comment:
minor nit:
this file is supposed to be sorted in recency order, so this new paragraph would appear above existing "Non-zeroes reserved bits" paragraph.